use-timescale/continuous-aggregates/about-continuous-aggregates.md

```diff
@@ -17,6 +17,88 @@ import CaggsTypes from "versionContent/_partials/_caggs-types.mdx";
 
 <CaggsTypes />
 
+## Components of a continuous aggregate
+
+Continuous aggregates consist of:
+
+*   Materialization hypertable to store the aggregated data in
+*   Materialization engine to aggregate data from the raw, underlying, table to
+    the materialization hypertable
+*   Invalidation engine to determine when data needs to be re-materialized, due
+    to changes in the data
+*   Query engine to access the aggregated data
+
+### Materialization hypertable
+
+Continuous aggregates take raw data from the original hypertable, aggregate it,
+and store the intermediate state in a materialization hypertable. When you query
+the continuous aggregate view, the state is returned to you as needed.
+
+Using the same temperature example, the materialization table looks like this:
+
+|day|location|chunk|avg temperature partial|
+|-|-|-|-|
+|2021/01/01|New York|1|{3, 219}|
+|2021/01/01|Stockholm|1|{4, 280}|
+|2021/01/02|New York|2||
+|2021/01/02|Stockholm|2|{5, 345}|
+
+The materialization table is stored as a Timescale hypertable, to take
+advantage of the scaling and query optimizations that hypertables offer.
+Materialization tables contain a column for each group-by clause in the query,
+a `chunk` column identifying which chunk in the raw data this entry came from,
+and a `partial aggregate` column for each aggregate in the query.
+
+The partial column is used internally to calculate the output. In this example,
```

Comment:
I like the example because it points you onto something you have to think about (like average on average is dangerous), but it's nothing TimescaleDB does automatically. When you create a continuous aggregate with an average aggregation, it'll store the average, no partial / intermediate data set. I think you confuse that with some hyperfunction aggregations like counter_agg which kind of work that way.